ansible_events/drools/durable_rules_engine.py

```diff
@@ -0,0 +1,230 @@
+import json
+import logging
+
+import requests
+
+_logger = logging.getLogger(__name__)
+
+
+class DurableRulesEngine:
+    __host = None
```

Comment:
Is there any specific reason to use double underscore prefix for these class attributes?